core/build.gradle

```diff
@@ -26,7 +26,8 @@ dependencies {
             project(':grpc-api').sourceSets.test.output,
             project(':grpc-testing'),
             project(':grpc-grpclb'),
-            libraries.guava_testlib
+            libraries.guava_testlib,
+            libraries.opencensus_impl
```

Comment:
`opencensus_impl` should be `testRuntimeOnly` dependency and shouldn't be needed for compiling.

Reply:
Will do when I get to a computer

Reply:
Looks like this can't be done because I need access to StatsComponentImpl to avoid mutating global state